test/e2e/shell.go

```diff
@@ -35,6 +35,14 @@ var (
 var _ = Describe("Shell", func() {
 
 	defer GinkgoRecover()
+
+	// A number of scripts only work on gce
```

Comment:
Can you please add gke as well? This works on gke as well as gce since both support gcloud compute ssh for the ssh-to-node function (although it isn't currently passing - we should look into that).